src/orchestration.d.ts

```diff
@@ -0,0 +1,311 @@
+  /**
+   * Submit a transaction on behalf of the remote accoutn for execution on teh remote chain.
+   * @param msgs - records for the transaction
+   * @returns void
+   */
+  executeEncodedTx: (msgs: EncodeObject[]) => Promise<void>;
+  /** deposit payment from zoe to the account*/
+  deposit: (payment: Payment) => Promise<void>;
+  /** get Purse for a brand to .withdraw() a Payment from the account */
+  getPurse: (brand: Brand) => Promise<Purse>;
+  /**
+   * Close the remote account
+   */
+  close: () => Promise<void>;
+  /* transfer account to new holder */
+  prepareTransfer: () => Promise<Invitation>;
+}
+
+export type BrandOrDenom = Brand | Denom;
+
+/**
+ * An object that supports high-level operations for an account on a remote chain.
+ */
+export interface OrchestrationAccount {
+  /** @returns the underlying low-level operation object. */
+  getChainAcccount: () => Promise<ChainAccount>;
+  /** @returns an array of amounts for every balance in the account. */
+  getBalances: () => Promise<Amount[]>;
+  /** @returns the balance of a specific denom for the account. */
+  getBalance: (denom: BrandOrDenom) => Promise<Amount | undefined>;
+
+  getDenomTrace: (
+    denom: string,
+  ) => Promise<{ path: string; base_denom: string }>;
+  /**
+   * @returns all active delegations from the account to any validator (or [] if none)
+   */
+  getDelegations: () => Promise<Delegation[]>;
+  /**
+   * @returns the active delegations from the account to a specific validator (or [] if none)
+   */
+  getDelegation: (validator: ValidatorAddress) => Promise<Delegation[]>;
+  /**
+   * @returns the unbonding delegations from the account to any validator (or [] if none)
+   */
+  getUnbondingDelegations: () => Promise<UnbodingDelegation[]>;
+  /**
+   * @returns the unbonding delegations from the account to a specific validator (or [] if none)
+   */
+  getUnbondingDelegation: (
+    validator: ValidatorAddress,
+  ) => Promise<UnbodingDelegation>;
+  getRedelegations: () => Promise<Redelegation[]>;
+  getRedelegation: (
+    srcValidator: ValidatorAddress,
+    dstValidator?: ValidatorAddress,
+  ) => Promise<Redelegation>;
+  /**
+   * Get the pending rewards for the account.
+   * @returns the amounts of the account's rewards pending from all validators
+   */
+  getRewards: () => Promise<Amount[]>;
+  /**
+   * Get the rewards pending with a specific validator.
+   * @param validator - the validator address to query for
+   * @returns the amount of the account's rewards pending from a specific validator
+   */
+  getReward: (validator: ValidatorAddress) => Promise<Amount[]>;
+  /**
+   * Transfer amount to another account on the same chain. The promise settles when the transfer is complete.
+   * @param toAccount - the account to send the amount to. MUST be on the same chain
+   * @param amount - the amount to send
+   * @returns void
+   */
+  send: (toAccount: ChainAddress, amount: Amount) => Promise<void>;
```

Comment:
_question_

What are the tradeoffs for using `Payment` instead of `Amount` here?

Reply:
It would require support of remote payments on other chains. That is something that we woudl like but is likely a layer above this.